source/our_tools/other_pipelines.rst

```diff
@@ -34,25 +34,24 @@ This produces a QC report for each important section of the tractoflow. For inst
 
 We will soon add here a list of characteristic you should expect to see for each metric, to help beginners develop an expert eye when looking at processed data.
 
-*How to share my work?*
 
-If you have reached this point, congratulations! You have run tractoflow, which uses a significant amount of computation time, and you have verified your data, which required a significant amount of your time. It would be great to ensure that other lab members don't need to reproduces the same steps for the same data. If you believe your data should be shared to the rest of the lab:
+QA/ QC  (Quality assurance, Quality check) -- recobundlesX
+----------------------------------------------------------
 
-    1. Please ask at least one other person to double-check your QC. You can share your html files on our slack channel #HelpMeQcMyData.
+To assess the quality of your bundles, the best way is to check the data visually using the mosaic, as described on the RecobundlesX page. However if you possess the ground truth for your data, you may use tractometry for your statistics.
 
-    2. Plase talk with Arnaud. He will tell you the best way to include your processed data either to BrainData or to Beluga.
 
-RecobundlesX
-------------
+Tractometry
+-----------
 
-ToDO
+If you have a ground truth, you can compute statistics for each bundle using the tractometer. You can find the definition of each metric (ex., VB, IB, VC, IC, OL, OR) on the `tractometer website <http://tractometer.org/ismrm_2015_challenge/evaluation>`_.
```

Comment:
We have a new scil_scoring or something like that, that Antoine just finished. Probably best to recommend using that then the tractometer evaluation script.